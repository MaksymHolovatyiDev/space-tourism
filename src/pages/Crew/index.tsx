import styles from './styled.module.scss';

import {CrewSlider} from 'widgets/crew-slider';

function Crew() {
  return (
    <main className={styles.root}>
      <CrewSlider />
    </main>
  );
}

export default Crew;
