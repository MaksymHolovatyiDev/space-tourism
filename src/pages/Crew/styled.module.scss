.root {
  height: 100vh;
  width: 100vw;

  display: flex;

  justify-content: flex-start;
  align-items: flex-end;

  padding-left: 165.73px;
  padding-bottom: 94px;

  background-size: cover;
  background-repeat: no-repeat;
  background-image: url('./assets/background-crew-desktop.jpg');
}
