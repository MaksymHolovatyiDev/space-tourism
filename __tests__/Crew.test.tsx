import renderer from 'react-test-renderer';
import Crew from '../src/pages/Crew';
import React from 'react';

it('renders correctly', () => {
  const tree = renderer.create(<Crew />).toJSON();
  expect(tree).toMatchSnapshot();
});
