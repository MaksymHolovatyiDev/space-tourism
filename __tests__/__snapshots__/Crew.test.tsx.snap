// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main>
  <div>
    <ul
      style={
        {
          "transform": "translateX( 0%)",
          "transition": "transform 250ms linear",
        }
      }
    >
      <li>
        <p
          className="undefined undefined undefined"
        >
          Commander 
        </p>
        <p
          className="undefined undefined undefined"
        >
          Douglas Hurley
        </p>
        <p
          className="undefined undefined"
        >
          Douglas Gerald Hurley is an American engineer, former Marine Corps pilot and former NASA astronaut. He launched into space for the third time as commander of Crew Dragon Demo-2.
        </p>
        <img
          alt="Douglas Hurley"
          src="src/entities/crew-member/assets/image-douglas-hurley.webp"
          style={
            {
              "right": 137,
            }
          }
        />
      </li>
      <li>
        <p
          className="undefined undefined undefined"
        >
          Mission Specialist 
        </p>
        <p
          className="undefined undefined undefined"
        >
          MARK SHUTTLEWORTH
        </p>
        <p
          className="undefined undefined"
        >
          Mark Richard Shuttleworth is the founder and CEO of Canonical, the company behind the Linux-based Ubuntu operating system. Shuttleworth became the first South African to travel to space as a space tourist.
        </p>
        <img
          alt="MARK SHUTTLEWORTH"
          src="src/entities/crew-member/assets/image-mark-shuttleworth.webp"
          style={
            {
              "right": 186.56,
            }
          }
        />
      </li>
      <li>
        <p
          className="undefined undefined undefined"
        >
          PILOT
        </p>
        <p
          className="undefined undefined undefined"
        >
          Victor Glover
        </p>
        <p
          className="undefined undefined"
        >
          Pilot on the first operational flight of the SpaceX Crew Dragon to the International Space Station. Glover is a commander in the U.S. Navy where he pilots an F/A-18.He was a crew member of Expedition 64, and served as a station systems flight engineer. 
        </p>
        <img
          alt="Victor Glover"
          src="src/entities/crew-member/assets/image-victor-glover.webp"
          style={
            {
              "right": 142.61,
            }
          }
        />
      </li>
      <li>
        <p
          className="undefined undefined undefined"
        >
          Flight Engineer
        </p>
        <p
          className="undefined undefined undefined"
        >
          Anousheh Ansari
        </p>
        <p
          className="undefined undefined"
        >
          Anousheh Ansari is an Iranian American engineer and co-founder of Prodea Systems. Ansari was the fourth self-funded space tourist, the first self-funded woman to fly to the ISS, and the first Iranian in space. 
        </p>
        <img
          alt="Anousheh Ansari"
          src="src/entities/crew-member/assets/image-anousheh-ansari.webp"
          style={
            {
              "right": 89.43,
            }
          }
        />
      </li>
    </ul>
    <ul>
      <li>
        <button
          className="undefined undefined"
          onClick={[Function]}
          type="button"
        />
      </li>
      <li>
        <button
          className="undefined false"
          onClick={[Function]}
          type="button"
        />
      </li>
      <li>
        <button
          className="undefined false"
          onClick={[Function]}
          type="button"
        />
      </li>
      <li>
        <button
          className="undefined false"
          onClick={[Function]}
          type="button"
        />
      </li>
    </ul>
  </div>
</main>
`;
